index.js

```diff
@@ -162,6 +164,58 @@ const inventoryCommand = (message) => {
   });
 };
 
+const lookUpCommand = (message) => {
+  const originalMessage = message.content;
```

Comment:
creating variables to name things is good -- in this case we're only using `originalMesage` in one place, so it might be better to just delete it